seacrowd/sea_datasets/mkqa/mkqa.py

```diff
@@ -0,0 +1,246 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{longpre-etal-2021-mkqa,
+    title = "{MKQA}: A Linguistically Diverse Benchmark for Multilingual Open Domain Question Answering",
+    author = "Longpre, Shayne  and
+      Lu, Yi  and
+      Daiber, Joachim",
+    editor = "Roark, Brian  and
+      Nenkova, Ani",
+    journal = "Transactions of the Association for Computational Linguistics",
+    volume = "9",
+    year = "2021",
+    address = "Cambridge, MA",
+    publisher = "MIT Press",
+    url = "https://aclanthology.org/2021.tacl-1.82",
+    doi = "10.1162/tacl_a_00433",
+    pages = "1389--1406",
+}
+"""
+
+_DATASETNAME = "mkqa"
+
+_DESCRIPTION = """\
+Multilingual Knowledge Questions and Answers (MKQA), an open-domain question answering evaluation set comprising 10k question-answer pairs aligned across 26 typologically diverse languages (260k question-answer pairs in total)
+"""
+
+_HOMEPAGE = "https://github.com/apple/ml-mkqa"
+
+_LICENSE = Licenses.CC_BY_SA_3_0.value
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://github.com/apple/ml-mkqa/raw/main/dataset/mkqa.jsonl.gz",
+}
+
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+_LANGUAGES = [
+    "khm",
+    "msa",
```

Comment:
`"msa"` --> `"zlm"`

Reply:
> `"msa"` --> `"zlm"`

Done.
Just in case, can I confirm that the language for this particular dataset is `"zlm"`, and not to be confused with standard Malay `"zsm"` (official language of Malaysia)?

Reply:
Good question, @fhudi. 😂 

@yongzx, could you please advise us on the difference between `zlm` and `zsm`? Or are they (almost) interchangeable in terms of linguistics like `fil` and `tgl` (CMIIW @jcblaisecruz02)?

Reply:
just curious but why not use the `msa` as it is the macrolanguage code?

Reply:
Based on the [proposal of splitting the language codes](https://iso639-3.sil.org/sites/iso639-3/files/change_requests/2007/2007-183.pdf), `zlm` was initially proposed as Local Malay, disambiguating with the already institutionalised Standard Malay which later became `zsm` after the split.

However, the name "Local Malay" was not adopted with following explanation (taken from [comment of the proposal's outcome](https://iso639-3.sil.org/sites/iso639-3/files/change_requests/2007/CR_Comments_2007-183.pdf))
> The use of the name “Local Malay” as a name to be used with the code element [zlm] for the language
denoting these closely related varieties was felt to be prone to confusion with the term “local Malay” meaning
the genetic subgroup (as in the above paragraph), which is a wider grouping of languages that are clearly
distinguished from one another phonologically, morphologically, and lexically. It was therefore less desirable
to use the name “Local Malay” than to retain the (admittedly cumbersome) “Malay (individual language),”
since there is no single more precise name that aptly labels this group of largely intelligible Malay varieties,
which are typically called just “Malay” by their speakers. 

IMHO, this is pretty similar case with "Bahasa Gaul" for the Indonesian, used only in the Jakarta Greater Area.
(If the difference with `ind` could be considered as different language, we could propose the language as `jkt` 😅)

---
Referrence: https://iso639-3.sil.org/request/2007-183

Reply:
Yea so my 2 cents.

Malay has different variations: we have Kelantan Malay, Sarawak Malay, etc. (each of these dialects have their own finegrained language code). 

Local Malay can be very confusing because it doesn't specify which localized dialects or variations it is. Standard Malay code `zsm` should be adopted if we are using ISO 639-3. 

I've went through the raw dataset files. It looks like Standard Malay to me (although very informally written – no proper casing and appropriate punctuations. Code-mixed with English songs title. Urgh I guess that's the Natural questions indeed.)

Reply:
@holylovenia done. please check.